runtime/contract_update_test.go

```diff
@@ -685,6 +685,8 @@ func TestRuntimeContractRedeploymentInSeparateTransactions(t *testing.T) {
 func TestRuntimeLegacyContractUpdate(t *testing.T) {
 	t.Parallel()
 
+	t.SkipNow()
```

Comment:
Was this just temporary? Maybe leave a TODO comment

Reply:
This test along with the legacy contract update feature is removed now: https://github.com/onflow/cadence/pull/3826/commits/0e5c9afbd83f822f203cfe4e6ee2c17c3fd98709

Reply:
Ah, I should have looked at the test name, I didn't realize it was related to the legacy contract update feature 👍